### PR TITLE
Fix RSS feed fetch-indicators

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-40505

## Description
Fixed an issue in fetch-indicators where reports have invalid URL links.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedRSS/ReleaseNotes/1_0_10.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### RSS Feed
+
+- Fixed an issue in ***fetch-indicators*** where reports have invalid URL links.
```
Comment (YuvHayun) [inline Packs/FeedRSS/ReleaseNotes/1_0_10.md:6]:
```suggestion
- Fixed an issue in ***fetch-indicators*** where reports failed due to invalid URL links.
```

**Review by YuvHayun — APPROVED** [review]
Nice!

File: Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ def create_indicators_from_response(self):
                     continue
                 published_iso = published.strftime('%Y-%m-%dT%H:%M:%S')
                 publications.append({
-                    'timestamp': indicator.get('published'),
+                    'timestamp': published_iso,
```
Comment (dantavori) [inline Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py:54]: why was this changed?

File: Packs/FeedRSS/ReleaseNotes/1_0_10.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
```
Comment (dantavori) [inline Packs/FeedRSS/ReleaseNotes/1_0_10.md:6]:
```suggestion
- Fixed an issue where the ***fetch-indicators*** command failed while processing report indicators with invalid URLs.
```

**Review by dantavori — APPROVED** [review]

File: Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py (hunk 2 above)
Comment (yaakovpraisler) [reply Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py:54]: Because there are instances where the 'published` is in a format that the server doesn't support and raises the following error:
```
Error from create indicators response: [400] Failed with error: [1:1213] failed to parse field [CustomFields.publications_grid.timestamp_date] of type [date] in document with id '2d6667d68d3028bd682e2a5a19b792aa'. Preview of field's value: 'Fri, 8 Mar 2019 19:45:00 +0000'. Other reasons: [[{illegal_argument_exception failed to parse date field [Fri, 8 Mar 2019 19:45:00 +0000] with format [basic_date_time||strict_date_optional_time||epoch_millis||yyyy-MM-dd HH:mm:ssz||strict_date_optional_time_nanos||yyyy-MM-dd HH:mm:ss.SSSSSSz||yyyy-MM-dd HH:mm:ss]}]]
```

**Comment by yaakovpraisler** [comment]
Build fails on docker issues that are not related to the PR changes, and coverage fails because it's not reaching 70% but my test increased the coverage from 52% to 55%.
@dantavori Can we have here a force merge?
